mapbox_gl_platform_interface/lib/src/my_location_style.dart

```diff
@@ -0,0 +1,10 @@
+part of mapbox_gl_platform_interface;
+
+class MyLocationStyle {
+  final String? puckColor;
+  final String? pulsingColor;
+
+  const MyLocationStyle({this.puckColor, this.pulsingColor});
+
+  dynamic toJson() => <dynamic>[puckColor, pulsingColor];
```

Comment:
it would be better to convert this to a map {"puckColor": puckColor, "pulsingColor": pulsingColor}